src/test/java/io/personium/core/bar/BarFileExporterTest.java

```diff
@@ -0,0 +1,322 @@
+import java.util.Map;
+import java.util.UUID;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathFactory;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.odata4j.edm.EdmEntitySet;
+import org.odata4j.producer.EntitiesResponse;
+import org.odata4j.producer.Responses;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Document;
+import org.xml.sax.InputSource;
+
+import io.personium.core.PersoniumUnitConfig;
+import io.personium.core.auth.AccessContext;
+import io.personium.core.model.Box;
+import io.personium.core.model.BoxRsCmp;
+import io.personium.core.model.Cell;
+import io.personium.core.model.CellRsCmp;
+import io.personium.core.model.DavCmp;
+import io.personium.core.model.ModelFactory;
+import io.personium.core.model.impl.es.CellEsImpl;
+import io.personium.core.model.impl.es.odata.CellCtlODataProducer;
+import io.personium.core.model.impl.fs.DavCmpFsImplTest.MockDavCmpFsImpl;
+import io.personium.core.model.jaxb.Ace;
+import io.personium.core.model.jaxb.Acl;
+import io.personium.core.utils.TestUtils;
+import io.personium.core.utils.UriUtils;
+
+/**
+ * Unit tests for BarFileExporter class.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(ModelFactory.ODataCtl.class)
+public class BarFileExporterTest {
+    public static String CELL_URL;
+    public static String BOX_SCHEMA_URL;
+    public static final String BOX_NAME = "box";
+
+    public static byte[] barFileBytes;
+    public static List<ZipEntry> barZipEntryList;
+    public static Map<String, ZipEntry> barZipEntryMap;
+    public static Map<String, byte[]> barZipContentMap;
+    static Logger log = LoggerFactory.getLogger(BarFileExporterTest.class);
+
+    /**
+     * Set Personium Unit configuration for the testing.
+     *   Unit url = https://unit.example/
+     *   Path-based Cell URL = false
+     */
+    @BeforeClass
+    public static void beforeClass() {
+        PersoniumUnitConfig.set(PersoniumUnitConfig.PATH_BASED_CELL_URL_ENABLED, "false");
+        // CommonUtils.setFQDN("unit.example");
+        PersoniumUnitConfig.set(PersoniumUnitConfig.UNIT_PORT, "");
+        PersoniumUnitConfig.set(PersoniumUnitConfig.UNIT_SCHEME, "https");
+        CELL_URL = UriUtils.convertSchemeFromLocalUnitToHttp("personium-localunit:user1:/");
+        BOX_SCHEMA_URL = UriUtils.convertSchemeFromLocalUnitToHttp("personium-localunit:app1:/");
```

Comment:
Actual value is like `https://demo.personium.io/app-personium-calendar/`. It's better to use this value.